backend/app/services/openapi/chat_response.py

```
@@ -0,0 +1,556 @@
+            task_crd.status.status = "FAILED"
+            task_crd.status.errorMessage = error_message
+            task_crd.status.updatedAt = datetime.now()
+            setup.task.json = task_crd.model_dump(mode="json")
+            setup.task.updated_at = datetime.now()
+            flag_modified(setup.task, "json")
+            db.commit()
+
+        raise HTTPException(
+            status_code=status.HTTP_500_INTERNAL_SERVER_ERROR,
+            detail=f"LLM request failed: {error_message}",
+        )
```

Ruff (0.14.10)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)